{"uuid":"2aab0363-88cf-43b3-87d0-c425f7fcf989","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["494ab63f-752d-425f-be77-1fdb913e9b4a"],"befores":[{"name":"appHooks.Hooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673417477825,"stop":1673417477825}],"afters":[{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673417477825,"stop":1673417477825},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673417478226,"stop":1673417478226},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673417478367,"stop":1673417478367},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"05d130ce-6c6b-4ed3-bdab-36782d1f2211-attachment","type":"image/png"},{"name":"Myscreenshot","source":"0d42481f-b767-4ae4-ab0e-8d871a6037c3-attachment"}],"parameters":[],"start":1673417478381,"stop":1673417478724}],"start":1673417477825,"stop":1673417478725}